pinecone/client_test.go

```diff
@@ -629,11 +275,8 @@ func (ts *IntegrationTests) TestConfigureIndexHitPodLimit() {
 	require.ErrorContainsf(ts.T(), err, "You've reached the max pods allowed", err.Error())
 }
 
-func (ts *IntegrationTests) deleteIndex(name string) error {
-	return ts.client.DeleteIndex(context.Background(), name)
-}
-
-func (ts *IntegrationTests) TestExtractAuthHeader() {
+// Unit tests:
```

Comment:
Make sure to remove the `// Unit tests:` line below -- I think it's here twice now